pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -168,7 +168,7 @@ func (r *ReconcileApplicationMonitoring) CreatePrometheusCRs(cr *applicationmoni
 		return reconcile.Result{Requeue: true}, err
 	}
 
-	for _, resourceName := range []string{PrometheusServiceAccountName, PrometheusServiceName, PrometheusRouteName, PrometheusCrName} {
+	for _, resourceName := range []string{PrometheusServiceAccountName, PrometheusServiceName, PrometheusCrName} {
```

Comment:
@davidkirwan where is the route created when you remove it from here?

Reply:
https://github.com/integr8ly/application-monitoring-operator/blob/a9bd35819053746cf89f0d1ad5e9cef6b2460da4/pkg/controller/applicationmonitoring/applicationmonitoring_controller.go#L162 in here. Looks like it was in there twice?

Reply:
ah nice catch